Rocket.Chat.SDKTests/Managers/LivechatManagerTests.swift

```diff
@@ -12,6 +12,7 @@ import SwiftyJSON
 
 class LivechatManagerTests: XCTestCase {
 
+    // swiftlint:disable:next force_unwrapping
     let socket = WebSocketMock(url: URL(string: "http://doesnt.matter")!)
```

Comment:
Yes, we should do avoid disabling it in production. Since we all know it should unwrap successfully or there is something wrong with it, and this is tests anyway, I think it's okay to disable it for one exception.